lib/rubycritic/browser.rb

```diff
@@ -0,0 +1,22 @@
+require "rubycritic/configuration"
+require "rubycritic/platforms/linux"
+require "rubycritic/platforms/darwin"
+
+module Rubycritic
+  class Browser
+    attr_reader :browser_name
```

Comment:
Why not just "name"? The "browser_" seems unnecessary since it is scoped to browser already.